### ISSUE TITLE
Mul + Add quantization

### ISSUE THREAD

**Issue by spacycoder** [alibaba/TinyNeuralNetwork#364]
Hi, I have quantized a model to int8 and the converter produces this graph:
![mul_add](https://github.com/user-attachments/assets/53a14036-2228-4c73-a4a2-03a36d0dbf27)

Why does it dequantize before the Mul op, is this expected?

**Comment by spacycoder** [comment]
This reproduces it:

```python
import torch.nn as nn
import torch
from tinynn.graph.quantization.quantizer import PostQuantizer
from tinynn.converter import TFLiteConverter

class Dummy(nn.Module):

    def __init__(self):
        super().__init__()
        self.max_pool = nn.MaxPool2d(kernel_size=4, stride=4)
        self.batch_norm = nn.BatchNorm2d(256)
    def forward(self, x):
        x = self.batch_norm(self.max_pool(x))
        return x

def _main():
    dummy_input0 = torch.rand(1, 256, 60,  60).float()
    model = Dummy()

    ptq_config = {
        "backend": "qnnpack",
        "per_tensor": False,
        "disable_requantization_for_cat": True
    }
    quantizer = PostQuantizer(
        model, (dummy_input0,), work_dir="mul_add_model", config=ptq_config
    )

    ptq_model = quantizer.quantize()
    ptq_model(dummy_input0,)

    with torch.no_grad():
        ptq_model.eval()
        ptq_model.cpu()

        ptq_model = quantizer.convert(ptq_model)
        torch.backends.quantized.engine = quantizer.backend
        converter = TFLiteConverter(
            ptq_model,
            (dummy_input0,),
            "mul_add_model.tflite",
            fuse_quant_dequant=True,
            quantize_target_type="int8"
        )
        converter.convert()

if __name__ == '__main__':
    _main()
```

**Comment by spacycoder** [comment]
I get the same with mul -> softmax:
![mul_softmax](https://github.com/user-attachments/assets/c00bd650-3967-4fd0-95a1-3bafd758a07a)

conversion code:
```python
import torch.nn as nn
import torch
from tinynn.graph.quantization.quantizer import PostQuantizer
from tinynn.converter import TFLiteConverter

class Dummy(nn.Module):

    def forward(self, x: torch.Tensor):
        scale_factor = 2.0
        weights = torch.softmax((x * scale_factor), dim=-1)
        return weights

def _main():
    dummy_input0 = torch.rand(1, 8, 225, 225).float()
    model = Dummy()

    ptq_config = {
        "backend": "qnnpack",
        "per_tensor": False,
        "disable_requantization_for_cat": True
    }
    quantizer = PostQuantizer(
        model, (dummy_input0,), work_dir="softmax", config=ptq_config
    )

    ptq_model = quantizer.quantize()
    ptq_model(dummy_input0,)

    with torch.no_grad():
        ptq_model.eval()
        ptq_model.cpu()

        ptq_model = quantizer.convert(ptq_model)
        torch.backends.quantized.engine = quantizer.backend
        converter = TFLiteConverter(
            ptq_model,
            (dummy_input0,),
            "softmax.tflite",
            fuse_quant_dequant=True,
            quantize_target_type="int8",
            rewrite_quantizable=True
        )
        converter.convert()

if __name__ == '__main__':
    _main()

```

**Comment by peterjc123** [comment]
Ok, I'll take a look. We are back from a long vacation.

**Comment by peterjc123** [comment]
> For QATQuantizer/PostQuantizer, set `config={"set_quantizable_op_stats": True}`
Please refer to [this table](https://github.com/alibaba/TinyNeuralNetwork/blob/main/docs/quantization_support.md#extra-flags-for-translating-the-above-ops-to-quantized-tflite).

**Comment by spacycoder** [comment]
Nice, hope you had a nice vacation!

**Comment by spacycoder** [comment]
Thanks, the softmax example is quantized when using the `config={"set_quantizable_op_stats": True}` option. But the first example still doesn't run quantized. Is this expected?

**Comment by peterjc123** [comment]
@spacycoder There is no direct mapping for BatchNorm in TFLite, so we cannot translate quantized BatchNorm. The only thing we could do is to restore it to a floating-point operator.

**Comment by spacycoder** [comment]
Ok, thanks!
